blues/posedart.py

```diff
@@ -276,12 +284,12 @@ def poseMove(self, context=None, residueList=None):
 
 
     def findDart(self, particle_pairs=None, particle_weights=None):
-        """ 
+        """
         For dynamically updating dart positions based on positions
         of other particles.
         This takes the weighted average of the specified particles
         and changes the dartboard of the object
-        
+
         Arguments
         ---------
         particle_pairs: list of list of ints
```

Comment:
Ints of what? Atom indices? Doc strings should have enough info that I could actually use the function without having to read the rest of the code.